cylc/flow/task_pool.py

```diff
@@ -1350,8 +1356,8 @@ def filter_task_proxies(self, items):
                     bad_items.append(item)
         return itasks, bad_items
 
-    @classmethod
-    def _parse_task_item(cls, item):
+    @staticmethod
+    def parse_task_item(item):
```

Comment:
Yes, true.. I think I re-implemented that change because it was still up in the air whether graphql was also going to be in `cylc-flow`j, and the filters used this... However, graphql doesn't really add a lot to the workflows.. so I could mark it private again